.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Description of Changes
+<!-- Brief description or overview on what was changed in the PR -->
+
+### Rationale behind Changes
+<!-- Why were these changes made?  What problem does it solve / area does it improve? -->
+
+### Suggested Testing Steps
+<!-- If applicable, including examples you've already tested with / recommendations for how to test further is very helpful! -->
+
+<!--
+### Recommended PR Guidelines
+The following is a list of _general_ style recommendations that will make reviewing and merging easier:
+- Commit Messages
+  - Please try to prefix your commit message, indicating what area of the project was modified.
+    - For example `gsdx: message...`.
+    - Looking at the project's commit history will help with keeping prefixes consistent overtime, there is no strictly enforced list.
+  - Try to keep messages brief and informative
+  - Remove unnecessary commits and squash commits together when appropriate.
+    - If you are not familiar with rebasing, check out the following resources:
+      - CLI - https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history
+      - GUI (SourceTree) - https://www.atlassian.com/blog/sourcetree/interactive-rebase-sourcetree
+- Code Styling and Formatting
+  - Consult the style guide - https://github.com/tadanokojin/pcsx2/blob/coding-guide/pcsx2/Docs/Coding_Guidelines.md
+  - Run `clang-format` using the configuration file in the root of the repository
+    - Visual Studio Setup - https://devblogs.microsoft.com/cppblog/clangformat-support-in-visual-studio-2017-15-7-preview-1/
+    - IMPORTANT - if you are running `clang-format` on unrelated changes (ie. formatting an entire file), please do so in a separate commit.
```

Comment:
Maybe we can add if the file hasn't been clang formatted and would like to skip the formatting stick to how the rest of the file is formatted.